v3/cypress/e2e/slider.spec.ts

```diff
@@ -76,8 +76,10 @@ context("Slider UI", () => {
 
     toolbar.getRedoTool().click()
     c.getComponentTitle("slider").should("have.text", newSliderName)
-
-    c.closeComponent("slider")
+    c.getComponentTile("slider").then($tile => {
+      cy.wrap($tile).find("[data-testid=component-close-button]").click("top", { force: true })
+    })
+    // c.closeComponent("slider") //Change in component header height causes interference with variable value input
```

Comment:
Rather than replicating this workaround dozens of times, how about including it in `closeComponent`? Something like:
```typescript
  closeComponent(component: string, index = 0) {
    if (component === "slider") {
      // Change in component header height causes interference with variable value input
      c.getComponentTile("slider").then($tile => {
        cy.wrap($tile).find("[data-testid=component-close-button]").click("top", { force: true })
      })
    }
    else {
      this.selectTile(component, index)
      this.getCloseButton(component, index).click({ force: true })
    }
```